userguide/one_shot_object_detection/README.md

```diff
@@ -84,16 +124,16 @@ You can preview the synthetic images the toolkit generates by calling the `previ
 
 ```python
 synthetic_images = \
-tc.one_shot_object_detector.util.preview_synthetic_training_data(starter_images, 'label')
+tc.one_shot_object_detector.util._augmentation.preview_synthetic_training_data(starter_images, 'label')
```

Comment:
Do not document "_" functions in the UG. Please remove this section if it's not ready.